app/controllers/statistics_announcements_controller.rb

```diff
@@ -31,12 +21,4 @@ def expire_cache_for_index_on_next_announcement_expiry(announcements)
     time_to_releases = announcements.map {|ann| ann.release_date - Time.zone.now }.reject {|time_span| time_span <= 0 }
     expires_in((time_to_releases << Whitehall.default_cache_max_age).min)
   end
-
-  def expire_cache_for_show_on_release(announcement)
```

Comment:
I've added a story to begin using the "publish intents" feature:
https://trello.com/c/zMdu3dHe/343-use-publish-intents-on-statistics-announcements